Observer/AfterPaymentVerifyObserver.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Profibro\Paystack\Observer;
```

Comment:
Please change the namespace too in line with the vendor change.

A suggestion would be `Paystack\Paystack\Observer`

Reply:
This applies to all places where a namespace is used as well as the folder names